pkg/apis/core/validation/validation_test.go

```diff
@@ -732,6 +747,29 @@ func TestValidatePersistentVolumeClaim(t *testing.T) {
 				StorageClassName: &validClassName,
 			}),
 		},
+		"invalid-claim-zero-capacity": {
```

Comment:
Sorry for the delay. But this test case is added below
https://github.com/kubernetes/kubernetes/pull/55532/files#diff-4547664c983457f8f7077a63a5ae7699R874